specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/examples/CosmosDBSqlRoleDefinitionDelete.json

```diff
@@ -0,0 +1,35 @@
+{
+  "parameters": {
+    "roleDefinitionId": "87717168-47d4-4dde-9021-25f683f6ec9d",
+    "subscriptionId": "subId",
+    "resourceGroupName": "rg1",
+    "databaseAccountName": "contoso",
+    "api-version": "2020-06-01-preview"
+  },
+  "responses": {
+    "202": {
+      "body": {
+        "id": "/subscriptions/subId/resourceGroups/rg1/providers/Microsoft.DocumentDB/databaseAccounts/contoso/cosmosRoleDefinitions/87717168-47d4-4dde-9021-25f683f6ec9d",
+        "name": "87717168-47d4-4dde-9021-25f683f6ec9d",
+        "type": "Microsoft.DocumentDB/databaseAccounts/cosmosRoleDefinitions",
+        "properties": {
+          "roleName": "Contoso Finance Records Reader",
+          "type": "CustomRole",
+          "assignableScopes": [
+            "/subscriptions/subId/resourceGroups/rg1/providers/Microsoft.DocumentDB/databaseAccounts/contoso/dbs/contoso-sales",
+            "/subscriptions/subId/resourceGroups/rg1/providers/Microsoft.DocumentDB/databaseAccounts/contoso/dbs/contoso-purchases"
+          ],
+          "permissions": [
+            {
+              "dataActions": [
+                "Microsoft.DocumentDB/databaseAccounts/databases/read",
+                "Microsoft.DocumentDB/databaseAccounts/databases/collections/read"
+              ]
+            }
+          ]
+        }
+      }
+    },
+    "204": {}
```

Comment:
What's the 204 for?

Reply:
This is for when the delete has completed successfully but there is no content returned.